ct/myspeed.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/tteck/Proxmox/main/misc/build.func)
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# Co-Author: MickLesk (Canbiz)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+function header_info {
+clear
+cat <<"EOF"
+    __  ___      _____                     __
+   /  |/  /_  __/ ___/____  ___  ___  ____/ /
+  / /|_/ / / / /\__ \/ __ \/ _ \/ _ \/ __  / 
+ / /  / / /_/ /___/ / /_/ /  __/  __/ /_/ /  
+/_/  /_/\__, //____/ .___/\___/\___/\__,_/   
+       /____/     /_/                        
+	   
+EOF
+}
+header_info
+echo -e "Loading..."
+APP="MySpeed"
+var_disk="4"
+var_cpu="1"
+var_ram="1024"
+var_os="debian"
+var_version="12"
+variables
+color
+catch_errors
+
+function default_settings() {
+  CT_TYPE="1"
+  PW=""
+  CT_ID=$NEXTID
+  HN=$NSAPP
+  DISK_SIZE="$var_disk"
+  CORE_COUNT="$var_cpu"
+  RAM_SIZE="$var_ram"
+  BRG="vmbr0"
+  NET="dhcp"
+  GATE=""
+  APT_CACHER=""
+  APT_CACHER_IP=""
+  DISABLEIP6="no"
+  MTU=""
+  SD=""
+  NS=""
+  MAC=""
+  VLAN=""
+  SSH="no"
+  VERB="no"
+  echo_default
+}
+
+function update_script() {
+	header_info
```

Comment:
Indented too much! lol

Reply:
lol, damn Visual Studio code, sometimes it makes big tabs in shell scripts